src/index.html

```diff
@@ -1,12 +1,504 @@
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="is-active nav__link">
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samrtphoneAndPhone" class="nav__link">
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#laptopAndPC" 
+              class="nav__link" 
+              data-qa="nav-hover"
+              >LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablet" class="nav__link">
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#foto" class="nav__link">
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              VIDEO
+            </a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+    
+    <main class="main-content">
+      <article class="container-section">
+        <section class="section">
+          <div class="card">
+            <div class="card__item" data-qa="card">
+              <img 
+                src="#" 
+                alt="photo-imac" 
+                class="card__image">
+              <h3 class="card__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h3>
+              <p 
+                class="card__product-code"
+                >Product code: 195434
+              </p>
+              <div class="feedback">
+                <div class="list-stars">
+                  <img
+                    src="#"
+                    alt="star-feedback" 
+                    class="list-stars__star list-stars__star--active"
+                    >
+                  <img
+                    src="#"
+                    alt="star-feedback" 
+                    class="list-stars__star list-stars__star--active"
+                    >
+                  <img
+                    src="#"
+                    alt="star-feedback" 
+                    class="list-stars__star list-stars__star--active"
+                    >
+                  <img
+                    src="#"
+                    alt="star-feedback" 
+                    class="list-stars__star list-stars__star--active"
+                    >
+                  <img
+                    src="#"
+                    alt="star-feedback" 
+                    class="list-stars__star list-stars__star--inactive"
+                    >
+                </div>
```

Comment:
Please create empty blocks for the stars and set the picture in styles as background-image